.github/workflows/deployDemo.yml

```diff
@@ -56,3 +56,5 @@ jobs:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Deploy frontend app
         run: yarn run deploy-azure-${{ env.DEPLOY_ENV }}
```

Comment:
Just a general question: why are we interpolating the environment name when each environment gets its own workflow definition? I didn't go character by character, but it looks like these workflows vary only by trigger and by the `DEPLOY_ENV` env var.

Reply:
It was a low effort way for me to copy paste this workflow 7 times